### PR TITLE
Update README.md to include cmd descriptions

### PR DESCRIPTION
**Pull request by zztin**
Some options are not included in the original README.md such as the -g group_file function of somalier relate. Including the complete cmd line help message to enable easier searching.

- Detail changes:
  - Include all help messages from somalier subcommands
  - VCF section was merged into previous section after usage for .bcf
- Other changes that can be done:
  - Currently, the # Usage paragraph appeared twice. Could be merged into one.
  - somalier ancestry' help message remain as somalier pca. Could be changed to somalier ancestry for consistency.

### CODE REVIEW

**Review by brentp — CHANGES_REQUESTED** [review]
Let's keep find-sites and ancestry as just one-line mentions and link to wiki.
Those are less-used and as the README gets too long it's less likely people will read it.

**Comment by brentp** [comment]
I put an ancestry wiki here: https://github.com/brentp/somalier/wiki/ancestry

File: README.md [hunk 1]
```diff
@@ -90,20 +103,108 @@ Note that if a new sample is added to the cohort, it's only necessary to perform
...
+                             optional sample prefixes that can be removed to find identical samples. e.g. batch1-sampleA batch2-sampleA
+  -p, --ped=PED              optional path to a ped/fam file indicating the expected relationships among samples.
+  -d, --min-depth=MIN_DEPTH  only genotype sites with at least this depth. (default: 7)
+  --min-ab=MIN_AB            hets sites must be between min-ab and 1 - min_ab. set this to 0.2 for RNA-Seq data (default: 0.3)
+  -u, --unknown              set unknown genotypes to hom-ref. it is often preferable to use this with VCF samples that were not jointly called
+  -i, --infer                infer relationships (https://github.com/brentp/somalier/wiki/pedigree-inference)
+  -o, --output-prefix=OUTPUT_PREFIX
+                             output prefix for results. (default: somalier)
+```
+### somalier ancestry (somalier pca)
```
Comment (zztin) [inline README.md]: help message for `somalier ancestry` and `somalier find-sites` are removed in the new commit 57c89de.

**Comment by zztin** [comment]
The README.md is updated as we discussed. It's ready for review.

**Pull request merged**